contrib/node/tests/python/pants_test/contrib/node/subsystems/test_package_managers.py

```diff
@@ -0,0 +1,176 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import mock
+import unittest
+
+from pants.contrib.node.subsystems.package_managers import (
+  PackageInstallationTypeOption, PackageInstallationVersionOption,
+  PackageManagerNpm, PackageManagerYarnpkg)
```

Comment:
I've been just making my python import lines really long, this style is much better and I will definitely do this from now on.